src/main/java/io/jenkins/plugins/orka/AgentTemplate.java

```diff
@@ -137,7 +142,13 @@ public String getRemoteFS() {
     public OrkaProvisionedAgent provision() throws IOException, FormException {
         this.ensureConfigurationExist();
         String vmName = this.createNewVMConfig ? this.configName : this.vm;
+        
+        logger.log(Level.INFO, "Deploying VM with name " + vmName);
```

Comment:
You can use `logger.info` directly.  
Additionally, this should be `Level.Fine`. It serves as a tracing log.  
We do not to overload the Jenkins info log with this information.  
This goes for all of the logging in the PR.